js/common.js

```diff
@@ -523,13 +523,10 @@ $(document).ready(function () {
     };
 
     const closeOffCanvasByClickOutside = (event) => {
-        if (
-            !offcanvasElement.contains(event.target) &&
-            !menuButton.contains(event.target)
-        ) {
-            closeOffcanvasNav()
+        if (event.target === backdropElement) {
+            closeOffcanvasNav();
         }
-    };
+    }
```

Comment:
Should this semicolon be retained?

```suggestion
    };
```

Reply:
Yes! Some other lines were also missing semicolons, so I just ran this navbar section through Prettier again.

Reply:
Maybe we should employ a linter?